pkg/cloud-provider/cloudapi/aws/aws_ec2.go

```diff
@@ -63,6 +66,7 @@ func newEC2ServiceConfig(name string, service awsServiceClientCreateInterface) (
 		resourcesCache:  &internal.CloudServiceResourcesCache{},
 		inventoryStats:  &internal.CloudServiceStats{},
 		instanceFilters: make(map[string][][]*ec2.Filter),
+		pollVpc:         true,
```

Comment:
I want to enable inventory poll for ec2 service even when filters are not set, this is how i am achieving it, instead of hardcoding it as "true" in NeedPollWithoutFilter() , i am using this variable.